### ISSUE TITLE
Why is 'StyleSheetApplied' set to 'false' on 'Browser_DocumentCompleted'?

### ISSUE THREAD

**Issue by RadarNyan** [andanteyk/ElectronicObserver#84]
I've been trying to eliminate flickers of the Browser, it's not a bug that breaks things, it just annoys me. After reading the code, at first I thought the flickers come from 74EO applying stylesheet to too many elements, so I reduced it to [as less as possible](https://github.com/RadarNyan/ElectronicObserver/commit/284b3fed4681d04cdd051cbdc29a4ec2a4fe0422) (at least that's the shortest answer I can come up with).

But the flickers can still be seen when the page is loading slow. Then I found [this](https://github.com/andanteyk/ElectronicObserver/blob/b0c1edcb9fd911d1b835052466ee8ff936b62f3b/Browser/FormBrowser.cs#L48-L68) :

``` c#
        private bool StyleSheetApplied {
            get { return _styleSheetApplied; }
            set {

                if ( value ) {
                    //Browser.Anchor = AnchorStyles.None;
                    ApplyZoom();
                    SizeAdjuster_SizeChanged( null, new EventArgs() );

                } else {
                    SizeAdjuster.SuspendLayout();
                    //Browser.Anchor = AnchorStyles.Top | AnchorStyles.Left;
                    Browser.Location = new Point( 0, 0 );
                    Browser.MinimumSize = new Size( 0, 0 );
                    Browser.Size = SizeAdjuster.Size;
                    SizeAdjuster.ResumeLayout();
                }

                _styleSheetApplied = value;
            }
        }
```

and it's called from [here](https://github.com/andanteyk/ElectronicObserver/blob/b0c1edcb9fd911d1b835052466ee8ff936b62f3b/Browser/FormBrowser.cs#L273) :

``` c#
        private void Browser_DocumentCompleted( object sender, WebBrowserDocumentCompletedEventArgs e ) {

            StyleSheetApplied = false; // <- line 273
```

So every time Browser_DocumentCompleted is triggered after the stylesheet is applied (it could happen due to links in other js files loading slower than the game - that could happen a lot), a flicker will happen.

Now here comes the question: what is this [line 273](https://github.com/andanteyk/ElectronicObserver/blob/b0c1edcb9fd911d1b835052466ee8ff936b62f3b/Browser/FormBrowser.cs#L273) for? I'd like to [delete it](https://github.com/RadarNyan/ElectronicObserver/commit/9dbd8bac2a14270b5f2c460e8d21672107255c2d) to avoid Browser flickering.

**Comment by andanteyk** [comment]
It seems to be a code that is from the first commit...
Perhaps, I think that to try to re-apply the stylesheet when you transition to another page.

If you delete it and bug does not occur, I think that it is safe to remove.
(Rather, the current behavior might be a bug...)

**Comment by andanteyk** [comment]
resolved by #85.
Thank you for your cooperation.
